packages/language-server/src/MessageHandler.ts

```diff
@@ -63,11 +63,10 @@ import {
 
 export async function handleDiagnosticsRequest(
   document: TextDocument,
-  binPath: string,
   onError?: (errorMessage: string) => void,
 ): Promise<Diagnostic[]> {
   const text = document.getText(fullDocumentRange(document))
-  const res = await lint(binPath, text, (errorMessage: string) => {
+  const res = lint(text, (errorMessage: string) => {
```

Comment:
Is the change from awaiting to not any more a thing we should think about in any way? @tomhoule

Reply:
I'm not a node expert, but the only risk I see with sync code is if takes too long before returning. I don't think it's a problem for prisma-fmt. We could make the rust code async with extra work, but I don't think this has any upside until we have long-running operations in prisma-fmt.